### ISSUE TITLE
Retrieve device AccessToken using Rules chains

### ISSUE THREAD

**Issue by NicoWii4** [thingsboard/thingsboard#5728]
**Component**

-  Rule Engine

**Description**
Hi,
I want to retrieve my device AccessToken using rule chains. I can not get this token using "Originator Field" node or other (tried to get "credentialsId", etc... but it does not work).

I found another solution but it doesn't seem to be the right one:
I use the swagger API to get my credential (Rest API node with GET "https://MY_THINGSBOARD/api/MY_DEVICEID/credentials") but I need a JWT_TOKEN to authorize this.
![get_credential](https://user-images.githubusercontent.com/80882254/146169454-ac7a1d2d-cd94-43fe-b9b7-816110f04f61.png)

My JWT_TOKEN is refreshed every 2.5 hours (I want to keep this) so I retrieve the current token using API (Rest API node with POST https://MY_THINGSBOARD/api/auth/login")
![get_jwt](https://user-images.githubusercontent.com/80882254/146169494-7ed8793b-3244-4f8e-af27-ceb622ef9804.png)

To success the node above, I need to know the username and password according to the Thingsboard doc [https://thingsboard.io/docs/reference/rest-api/](url). So I build a message with a fixed username and a fixed password (a tenant admin created before) but it seems to me that it's not the right thing to do since it can break my Thingsboard security. 

To sum up, I have the following questions:

- Can I retrieve easily device AccessToken with a rule chain?
- If I need to use API, can I easily retrieve current JWT_TOKEN? Can I retrieve current customer information (username and password)?

**Environment**

- OS:  Linux (Ubuntu 20.04)
- ThingsBoard: 3.3.1 PE
- Browser: Mozilla 95.0

Thank you in advance for your response,
Best regards

**Comment by bensonzhow** [comment]
There is no other good way except building a login rest api through username and password, or customizing nodes to obtain JWT_TOKEN. Note that obtaining JWT_TOKEN through rest api login will generate a lot of login audit logs.
<img width="757" alt="image" src="https://github.com/thingsboard/thingsboard/assets/1768516/7b4c8489-49bb-451e-a3b1-642951a1d531">
<img width="1267" alt="image" src="https://github.com/thingsboard/thingsboard/assets/1768516/45ca6081-98f6-40ff-86c5-9a63c5683180">

**Comment by mykhailokornieiev** [comment]
Closed due to inactivity
